src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -280,6 +290,14 @@ public void updateReplicateStatus(String chainTaskId,
             return Optional.empty();
         }
 
+        if (statusUpdate.getDetails() != null && statusUpdate.getDetails().getStdout() != null) {
+            if (statusUpdate.getStatus().equals(COMPUTED)) {
+                String stdout = statusUpdate.getDetails().tailStdout().getStdout();
```

Comment:
Do we tail here again in case a tweaked worker doesn't tail?

Reply:
Exactly!